submissions/nhunzaker/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "viget-university",
+  "version": "0.1.0",
+  "description": "This is the Viget internal workshop repo for the excellent HapiJS University workshop. This will differ slightly than the original.",
+  "main": "lib/index.js",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
+    "hapi": "12.0.0"
+  },
+  "devDependencies": {
+    "istanbul": "0.4.1",
+    "mocha": "2.3.4",
+    "standard": "5.4.1"
+  },
+  "scripts": {
+    "start": "node lib/index",
+    "lint": "standard lib/**/*.js",
```

Comment:
Have you noticed any rules defined in `standard` that conflict with your personal preferences?